testcase/erc721TransferTC/erc721TransferTC.go

```diff
@@ -60,28 +60,41 @@ func Init(accs []*account.Account, endpoint string, gp *big.Int) {
 
 func Run() {
 	cli := cliPool.Alloc().(*client.Client)
+	defer cliPool.Free(cli)
 
-	fromAcc := accGrp[rand.Intn(nAcc)]
 	toAcc := accGrp[rand.Intn(nAcc)]
 
-	// Get token ID from the channel
-	// Here is an assumption that it won't be blocked by the channel
-	// Although this go routine can be blocked, other can send a NFT to this account
-	fromNFTs := account.ERC721Ledger[fromAcc.GetAddress()]
-	tokenId := <-fromNFTs
+	// Find an account with available tokens
+	var fromAcc *account.Account
+	var tokenId *big.Int
+
+	// Try multiple accounts to find one with tokens
+	for i := 0; i < 200; i++ {
```

Comment:
It looks like we're trying to randomly pick an account with a token.
We could pick a random `number < len(accGrp)` and start iterate from the number, like:
```
start := rand.Intn(len(accGrp))
for i := range len(accGrp) {
    candidateAcc := accGrp[(start + i) % len(accGrp)]
```

That way we could assure that we try every account & pick random account if available?

Reply:
Applied https://github.com/kaiachain/kaia-load-tester/pull/22/commits/eaf1832767e00434cc6f77173fc70f6530bd5c0c. Btw, it would be so big if we iterate every account so I limit it by 200. Sometimes the array could be millions.